nevergrad/functions/base.py

```diff
@@ -204,6 +204,36 @@ def update_leaderboard(identifier: str, loss: float, array: np.ndarray, verbose:
         pass  # better avoir bugs for this
 
 
+class ArrayExperimentFunction(ExperimentFunction):
+    """Adds a "symmetry" parameter, which allows the creation of many symmetries of a given function."""
+
+    def symmetrized_function(self: AEF, x: tp.Any):
```

Comment:
```suggestion
    def symmetrized_function(self, x: np.ndarray) -> tp.Loss:
```

`AEF` is only necessary in very specify cases (typically for copy constructors, which return the same type as `self` while you don't necessary know the type of `self` since it can be a subclass)